docs/security.rst

```diff
@@ -52,23 +59,52 @@ By default Schema Registry ships with configs that allow you to make REST API ca
   * Importance: high
 
 ``schema.registry.inter.instance.protocol``
-  The protocol used while making calls between the instances of schema registry. The slave to master node calls for writes and deletes will use the specified protocol. The default value would be `http`. When `https` is set, `ssl.keystore.` and `ssl.truststore.` configs are used while making the call.
+  The protocol used while making calls between the instances of Schema Registry. The slave to master node calls for writes and deletes will use the specified protocol. The default value would be `http`. When `https` is set, `ssl.keystore.` and `ssl.truststore.` configs are used while making the call.
 
   * Type: string
   * Default: "http"
   * Importance: low
 
-If you would want to upgrade to HTTPS in an existing cluster, you need to perform the outlined steps:
+On the client, configure the ``schema.registry.listener`` to match the configured Schema Registry listener.
```

Comment:
``schema.registry.url`` instead of ``schema.registry.listener``

Reply:
Good catch.  Fixed.